Project-1/README.md

```diff
@@ -117,7 +117,7 @@ After attending two group homework sessions and working with Austin, I do not un
   ```
 
 
-5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both 
```

Comment:
Question 3. (a) B(x,y) <--> ∃x∃y∃z((Pzx ∧ Pzy) ∧ x ≠ y ∧ ~Fx)

Note: Here we are defining B(x,y) but not ∀x∀yB(x,y). In B(x,y), x and y are free variables so your definition of it should not use quantifers to bound x and y. So "∃x∃y" in (a)~(c) and "∃x" in (d)~(e) should be deleted. Also, if you wanna be careful enough, you may follow @Finn1928 by adding more non-identity statements (one cannot be a parent or brother of oneself.)

(e) You should tell z, w, r are male. So the definition should be: ∃y∃z∃w(¬(w=z)∧¬(x=w)∧¬(x=z)∧P(y,x)∧P(y,z)∧P(y,w)∧¬F(z)∧¬F(w))∧∀r(P(y,r)∧¬F(r)→ r=x ∨ r=z ∨ r=w)

Q4. I revise Ali and Giacomo's new approach and results are for reference here:

(a) B that says that x is a brother of y
  Person ≡ M ⊔ ¬M
  **p2** (parent of at least 2 children) ≡ ≥2 ∃parent_of.Person
  The inverse of p2 means that x is one of at least 2 children produced by y.
  B (brother) ≡ M ⊓ ∃p2¯.Person
(b)  A that says that x is an aunt of y
  
  A ≡ ¬M ⊓ (∃p2¯.(≥ 2 parent_of. (∃parent_of. Person)) ⊔ ¬∃parent_of. Person)
(c)  C that says that x and y are cousins
  **gp2** (Grandparent with at least two children which each have children) ≡ ≥2 parentof. (∃parentof. Person)
  C ≡ gp2¯. Person
  But this would only describe one of the two cousins, i.e. "being a cousin of someone" or "having a cousin". We cannot define "being cousins" in description logic.
(d)  O that says that x is an only child  
  **parent_only** (be a parent of exactly one child) ≡ (≥1 parent_of. Person) ⊓ (≤1 parent_of. Person)
O ≡ ∃parent_only¯.Person

(e)  T that says that x has exactly two brothers. 
  **p3m** (be a parent of exactly three male children) ≡ (≥3 parent_of. Male) ⊓ (≤3 parent_of. Male)
  **p1** (be a parent of at least one child) ≡ ≥1 parent_of. Person -- the inverse of p1 is just "be a child of"!
  T ≡ ∃p3m¯.Person ⊔ (¬M ⊓ ∃p1¯. (≥2 parent_of. Male ⊓ ≤2 parent_of. Male)) 
-- This means, either x is one of exactly 3 boys produced by a person (so x has exactly two brothers), or x is female and is a child of a person who has exactly two boys (so x has also exactly two brothers)

Q5. K = (T, A), T is a set including 2 formulas from Q4 and 1 or 2 new formula like "Aunt $\sqsubseteq$ ~Male", A is a set of specific statements like "Mary: Female". And then you give a model by giving a domain set and assigning some subset of the domain to each concept in K.

Q7. The key is to notice there are 2 "∀" in the first formula but there is only one "∀" in the second formula. 2 "∀" in the first formula requires that all points are connected to each other in a graph, but such a requirement is not present in the the second formula. Thus, you can draw a graph where some point (say, a) is not connected to all points including itself, although every point is connected to some point.

Q9. (a) 

(2) ¬(¬Px ∨ ¬Qx)	assumption
(3) (Px ∧ Qx)		2, negated disjunction

This move from (2) to (3) has used DeMorgan's law. So you are using DeMorgan's Law to prove itself, begging the question.

Q10. My answer is for reference:

Answer: 
For (a), a Fitch-style natural deduction proof relies on deductive rules like disjunction elimination or double negation elimination (DNE) to a more substantive degree than a tree proof. 
While a Fitch-style natural deduction proof is a **combinational application of different means of derivation**, a tree proof is essentially **an application of indirect derivation**. That is, when we do a tree proof, we have to decompose the target (complex) formula into its subformulas, and then differently connect negations of those subformulas in order to form different paths. Once when all paths somehow involve a contradiction, then the opposite possibilities of the target formula are all shown to be implausible, and thus the target formula will be proved.
Moreover, a Fitch-style natural deduction proof is to apply rules to the target formula and its logical consequents, while a tree proof does not apply the same rules but concentrates on finding possible logical contradictions among negated subformulas of the target formula.

Reply:
@katiejohnso lol YOU should consider Josh's comment to Question 9. I send you some pictures of Q9, which are complete unfolding proofs.

Reply:
Thank you so much for your thorough feedback, particularly with number four.  I am not confident in description logic whatsoever.